### DISCUSSION TITLE
minecraft path

### DISCUSSION THREAD

**Discussion by LRCaetanoM** [Kron4ek/minecraft-vortex-launcher#82] [Q&A]
Where can I find the minecraft path from vortex launcher so I can install OptiFine? 

![image](https://github.com/Kron4ek/minecraft-vortex-launcher/assets/95530995/1e0afe04-8143-4bc6-9444-5ac3bd1770b3)

Linux

**Comment by Kron4ek** [comment]
Vortex Launcher downloads the game in the same directory where the launcher itself resides.

**Comment by armjex** [comment]
> Where can I find the minecraft path from vortex launcher so I can install OptiFine?
> 
> ![image](https://private-user-images.githubusercontent.com/95530995/280100200-1e0afe04-8143-4bc6-9444-5ac3bd1770b3.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njg4NDE1MzYsIm5iZiI6MTc2ODg0MTIzNiwicGF0aCI6Ii85NTUzMDk5NS8yODAxMDAyMDAtMWUwYWZlMDQtODE0My00YmM2LTk0NDQtNWFjM2JkMTc3MGIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMTE5VDE2NDcxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkYjViNmE5YTgxZjcwMzJkZDgxMjYyNDAyYTZjMGIzYzgxNGNmMjBiYmNlYTVlOTRlNTRmNGVjMDJhOGUyNjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.akcz4n8X0KmQrS1pUErm8ZkrD2kf5bywe9Gf_x5MMQI)
> 
> Linux

I think you need the correct version that exactly matches the number.
